internal/server/singleprocess/poll_test.go

```diff
@@ -668,7 +668,7 @@ func TestApplicationPollHandler_turnoff(t *testing.T) {
 		raw, err := testServiceImpl(impl).state.JobList()
 		for _, j := range raw {
 			if j.State != pb.Job_ERROR &&
-				j.SingletonId == fmt.Sprintf("appl-poll/%s", appName) {
+				j.SingletonId == fmt.Sprintf("appl-poll/%s/deployment", appName) {
```

Comment:
We should add a new test for `TestApplicationPollHandler` to ensure if we have a deployment _and_ release, we return and queue two jobs and then a test with _just_ a deployment and we queue 1 job that's a deployment.